docs/core/upgrade.md

```diff
@@ -64,20 +50,40 @@ Migrations are run inside of an `UpgradeHandler` using  `app.mm.RunMigrations(ct
 
 ```go
 cfg := module.NewConfigurator(...)
-app.UpgradeKeeper.SetUpgradeHandler("my-plan", func(ctx sdk.Context, plan upgradetypes.Plan, vm module.VersionMap) (module.VersionMap, error) {
+app.UpgradeKeeper.SetUpgradeHandler("my-plan", func(ctx sdk.Context, plan upgradetypes.Plan, fromVM module.VersionMap) (module.VersionMap, error) {
 
     // ...
     // do upgrade logic
     // ...
 
-    // RunMigrations returns the VersionMap
-    // with the updated module ConsensusVersions
-    return app.mm.RunMigrations(ctx, vm)
+    // returns a VersionMap with the updated module ConsensusVersions
+    return app.mm.RunMigrations(ctx, fromVM)
 })
 ```
 
 To learn more about configuring migration scripts for your modules, see the [Module Upgrade Guide](../building-modules/upgrade.md).
 
+### Order Of Migrations
+
+All migrations are run in alphabetical order, except `x/auth` which is run the last. [hack] If you want to change the order of migration then you can run migrations in multiple stages. For example, you want to run `foo` last:
```

Comment:
"[hack]" seems weird in docs. Maybe something like:

```suggestion
All migrations are run in alphabetical order, except `x/auth` which is run the last, because of state dependencies between modules (you can read more in [issue #10606](https://github.com/cosmos/cosmos-sdk/issues/10606)). If you want to change the order of migration then you can run migrations in multiple stages. **Please beware that this is hacky, and make sure you understand what you are doing before running such migrations in production.** For example, you want to run `foo` last:
```

Reply:
I'm going to use emphasis instead of bold.